docs/Basic-Guide.md

```diff
@@ -22,26 +20,43 @@ for more information.
 4. Go to **Edit** > **Project Settings** > **Player**
 5. For **each** of the platforms you target (**PC, Mac and Linux Standalone**,
    **iOS** or **Android**):
-    1. Option the **Other Settings** section.
+    1. Expand the **Other Settings** section.
     2. Select **Scripting Runtime Version** to **Experimental (.NET 4.6
        Equivalent or .NET 4.x Equivalent)**
 6. Go to **File** > **Save Project**
 
+## Setting up the Inference Engine
+
+We provide pre-trained models for all the agents in our all demo environments. 
+To be able to run those models, you'll first need to set-up the Inference 
+Engine.The Inference Engine is a general API to
+run neural network models in Unity that leverages existing inference libraries such 
+as TensorFlowSharp and Apple's CoreML. Since the ML-Agents Toolkit uses TensorFlow 
+for training neural network models, the output model format is TensorFlow and 
+the model files include a `.tf` extension. Consequently, you need to install 
+the TensorFlowSharp backend to be able to run these models within the Unity 
+Editor. You can find instructions 
+on how to install the TensorFlowSharp backend [here](Inference-Engine.md).
+Once the backend is installed, you will need to reimport the models : Right click
+on the `.tf` model and select `Reimport`.
+
```

Comment:
Also to align with the format of the Basic Guide, the steps to use Inference Engine should be described within this page instead of in another page, so that our user has a place to go through all of the steps in one place.

Reply:
In the ideal scenario maybe, but ML-Agents will not host any Inference Engine documentation. So we can only link to external docs.

Reply:
Just like how we did things with Tensorflowsharp, we were not hosting TensorFlowSharp, but we put the docs on how to use TensorFlowSharp on our own github.